source/code-snippets/usage-examples/findOne.ts

```diff
@@ -0,0 +1,40 @@
+import { FindOptions, MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+type imdb = {
+  rating: number;
+  votes: number;
+  id: number;
+};
+
+interface MovieSchema extends Document {
+  title: string;
+  imdb: imdb;
+}
```

Comment:
```suggestion
export interface Movie {
  title: string;
  year: number;
  released: Date;
  plot: string;
  type: 'movie' | 'tv' | 'web series';
  imdb: IMDB;
}
```

I think showing more variety of types makes it a little more realistic but understandable if you want to keep the example terse.

You don't have to `extends Document` here, that makes the schema support any key, which may be desirable sometimes but not for this example.

Reply:
Sounds good, thanks for the note. I'll implement the suggested change.

The rest of this is purely for my own edification, feel free to ignore:

Sorry if this is a silly question, but I saw in the [collection method code](https://github.com/mongodb/node-mongodb-native/blob/b67af3cd/src/db.ts#L323) this generic type which made me think that the type passed to collection had to extend document:

>  <TSchema extends Document = Document>

Based off of this [stack overflow post](https://stackoverflow.com/questions/56843790/typescript-generic-type-with-equal-operator-means) I interpreted this as: The default type is document, and if you provide a type, it must extend document. I was surprised that you didn't need to extend `Document` in the `Movie` interface. Any tips on how I am misunderstanding this line of code?